src/Features/VisualBasic/Portable/Structure/VisualBasicStructureHelpers.vb

```diff
@@ -98,8 +103,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Structure
                 autoCollapse As Boolean,
                 type As String,
                 isCollapsible As Boolean) As BlockSpan?
-            Return CreateBlockSpan(blockNode.Span, bannerText, autoCollapse,
-                                type, isCollapsible, isDefaultCollapsed:=False)
+            Return CreateBlockSpan(
+                blockNode.Span, GetHintSpan(blockNode),
```

Comment:
Why is span different here?  Why aren't we just updating everything to `blockNode`?

Also, what happens (for both C# and BB) if you write the attribute on the same line as the block?